pipeline/gitlab-ci/dmc.yml

```diff
@@ -0,0 +1,29 @@
+---
+build_dmc:
+  stage: docker_management_container
+  image: docker:git
+  services:
+    - name: docker:dind
+      alias: docker
+  variables:
+    # renovate: datasource=github-releases depName=docker-management-container
+    GITHUB_DMC_TAG: 3.1.0
```

Comment:
Can these variables be overridden, if they're defined here?
I _think_, they have to be defined at top-level, see: https://docs.gitlab.com/ee/ci/variables/

Reply:
Yes the variable can be overridden. I attache this to the README.MD